docs/rest.md

```diff
@@ -670,7 +670,11 @@ Sample input:
 ```javascript
 {
     “action”: “connect”,
-    “plug”: {“snap”: “canonical-pi2”,   “plug”: “pin-13”},
-    “slot”: {“snap”: “keyboard-lights”, “slot”: “capslock-led”}
+    “plugs”: {
+        {“snap”: “canonical-pi2”,   “plug”: “pin-13”},
+    },
+    “slots”: {
+        {“snap”: “keyboard-lights”, “slot”: “capslock-led”},
+    }
```

Comment:
Corrected (and elsewhere in rest.md) this is a side effect of copy-pasting from google docs.